Makefile

```
@@ -0,0 +1,29 @@
+lint_sprocket:
+	@echo "Running sprocket lint on all modules..."
+	@echo "Checking if sprocket is available..."
+	@if ! command -v sprocket >/dev/null 2>&1; then \
+		echo >&2 "Error: sprocket is not installed or not in PATH. Install sprocket (https://sprocket.bio/installation.html)"; \
+		exit 1; \
+	fi
+	@for dir in modules/*/; do \
+		if [ -d "$$dir" ]; then \
+			echo "Linting $$dir"; \
+			sprocket lint "$$dir"; \
+		fi; \
+	done
+
+lint_miniwdl:
+	@echo "Running miniwdl lint on all modules..."
+	@echo "Checking if uv is available..."
+	@if ! command -v uv >/dev/null 2>&1; then \
+		echo >&2 "Error: uv is not installed or not in PATH. Install uv (https://docs.astral.sh/uv/getting-started/installation/)"; \
+		exit 1; \
+	fi
+	@for file in modules/*/*.wdl; do \
+		if [ -f "$$file" ]; then \
+			echo "Linting $$file"; \
+			uv run --python 3.13.5 --with miniwdl miniwdl check "$$file"; \
```

```suggestion
			uv run --python 3.13 --with miniwdl miniwdl check "$$file"; \
```
While testing, I ran into issues due to the specific `--python 3.13.5` tag. The error message produced was:
```
error: No interpreter found for Python 3.13.5 in virtual environments, managed installations, or search path
```
It was pretty easy to fix, just needed to simplify down to `--python 3.13` and it went off without a hitch, but I wonder if this full version tag is overly restrictive?

---

Weird. `uv` should just download the missing python version if it doesn't have it. 

But yeah, I don't think it matters, so let's do the 3.13.